### PR TITLE
add Exponential learning rate decay

### PR DESCRIPTION
**Pull request by SeaOfOcean**
## What changes were proposed in this pull request?

add Exponential learning rate decay

## How was this patch tested?

unit test

### CODE REVIEW

**Comment by SeaOfOcean** [comment]
pr validate pass

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala [hunk 1]
```diff
@@ -453,6 +453,29 @@ object SGD {
   }
 
   /**
+   * [[Exponential]] is a learning rate schedule, which rescale the learning rate by
```
Comment (zhangxiaoli73) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala]: Why [[...]]

**Comment by qiuxin2012** [comment]
LGTM

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala (hunk 1 above)
Comment (SeaOfOcean) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala]: It can help recognize Exponential as class in IDE

**Comment by zhichao-li** [comment]
LGTM

**Pull request merged**